src/sentry/integrations/github/client.py

```diff
@@ -659,6 +666,32 @@ def get_blame_for_file(
 
         return ref.get("target", {}).get("blame", {}).get("ranges", [])
 
+    def get_blame_for_files(self, files: Sequence[SourceLineInfo]) -> Sequence[FileBlameInfo]:
+        file_path_mapping = generate_file_path_mapping(files)
+
+        try:
+            response = self.post(
+                path="/graphql",
+                data={"query": create_blame_query(file_path_mapping)},
+                allow_text=False,
+            )
+        except ValueError as e:
+            sentry_sdk.capture_exception(e)
```

Comment:
Use `logger.exception("Some message")` instead. It will log to sdout plus it allows you to pass an `extra={}` dictionary if you wish to (e.g. org.slug and others).